src/main/scripts/install.sh

```diff
@@ -14,7 +14,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-MAX_RETRIES=99
+MAX_RETRIES=299
```

Comment:
That is a big jump. Can you say why such a big jump is necessary?

Reply:
Good question. The reason of increasing `MAX_RETRIES` is that waiting until the ip address of the ingress resource is available needs longer time, which is observed during the testing.